src/revChatGPT/V3.py

```diff
@@ -149,7 +151,7 @@ def ask_stream(
                 "frequency_penalty": kwargs.get("frequency_penalty", self.frequency_penalty),
                 "n": kwargs.get("n", self.reply_count),
                 "user": role,
-                # "max_tokens": self.get_max_tokens(convo_id=convo_id),
+                "max_tokens": self.get_max_tokens(convo_id=convo_id),
```

Comment:
Not needed since it defaults to max

Reply:
True, but if this isn't included then the bot would continue to stream a response until 4096 tokens is reached:
max_count=200
![image](https://user-images.githubusercontent.com/26652420/223458059-445c52fb-6d64-4de1-9d3d-e9d010de09c9.png)


Where as with the change the users set `max_tokens` is respected by the API:
max_count=200
![image](https://user-images.githubusercontent.com/26652420/223457309-4922f2bd-63cd-4276-a689-6c12f62a444d.png)

I would think this the desired outcome, or am i missing something?

Reply:
I wrongly assumed users would want the full response.